Framework/CurveFitting/src/Algorithms/ConvolveWorkspaces.cpp

```diff
@@ -52,9 +52,8 @@ void ConvolveWorkspaces::exec() {
 
   // Cache a few things for later use
   const size_t numHists = ws1->getNumberHistograms();
-  const size_t numBins = ws1->blocksize();
   Workspace2D_sptr outputWS = boost::dynamic_pointer_cast<Workspace2D>(
-      WorkspaceFactory::Instance().create(ws1, numHists, numBins + 1, numBins));
+      WorkspaceFactory::Instance().create(ws1));
```

Comment:
I changed this because, if the input workspace was a point data workspace, the `setSharedX` below failed. This way the output workspace is created with the same X, Y, E sizes as the input workspace, which I think is correct.